driver-core/src/main/java/com/datastax/driver/core/Metrics.java

```diff
@@ -219,6 +229,15 @@ public Errors getErrorMetrics() {
     }
 
     /**
+     * Returns the total number of in flight requests to Cassandra hosts.
+     *
+     * @return The total number of in flight requests to Cassandra hosts.
+     */
+    public Gauge<Integer> getInFlightRequests() {
```

Comment:
+1, from my experience most metrics would be more useful if they were exposed per host, I'll keep this in mind for the next major revision.